modules/ci/commit0.module.yml

```diff
@@ -0,0 +1,24 @@
+module: 
+  title: "CI templates"
+  description: ""
+  author: ""
+  icon: ""
+  thumbnail: ""
+  readme: "" # defaults to README.md
+
+# Template variables to populate, these could be overwritten by the file spefic frontmatter variables
+templates: 
+  strictMode: true # will only parse files that includes the .tmpl.* extension, otherwise it will copy file 
```

Comment:
Not sure on the naming here, also wouldn't we just parse all the files listed by default, or maybe do a .commitignore file to tell it to not parse certain files that aren't needed. For example maybe the README.md in the folder could be ignored, but the rest copied over.

Reply:
Sorry I didn't see your message earlier. Are you talking just about the `strictMode`? Yah we probably don't need this just yet. I thought of it as a way to improve performance. For a really sparsely templated project this might be a speed up.